djangoproject/settings.py

```diff
@@ -61,7 +61,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [BASE_DIR.joinpath('djangoproject/templates')],
```

Comment:
stavi ovaj `templates` folder na nivou `djangoapp`, jer tebi je to konrentno vezano za tu aplikaciju

Reply:
Onda bi trebalo da ti radi i sa DIRS: [], jer ti je dole navedeno APP_DIRS: True gdje ce on zaparavo da pretrazi sve template sub-driektorijume na nivou svake od tvojih aplikacija i cache-ira ih sto je dobra stvar. https://docs.djangoproject.com/en/3.1/ref/settings/#app-dirs

Reply:
Ima smisla, ispravljam i to odmah